packages/components/.storybook/style.scss

```
@@ -1,2 +1,7 @@
-@import '@wordpress/components/build-style/style';
-@import '@automattic/calypso-color-schemes';
+@use '@wordpress/components/build-style/style';
+@use '@automattic/calypso-color-schemes';
+@use "@wordpress/base-styles/variables";
+
+body {
+	font-family: variables.$default-font;
```

Should we prefer to use `$font-family-body` if `$default-font` is [targeted for deprecation](https://github.com/WordPress/gutenberg/blob/9935804000c070072575f7d9fea76b5d56eca3f1/packages/base-styles/_variables.scss#L15)?

```suggestion
	font-family: variables.$font-family-body;
```

---

Good point, I'll apply the suggested change